net-misc/seafile-client/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST seafile-client-7.0.10.tar.gz 2368056 BLAKE2B 8b446a818cc3a238bd92831843f1991632e9417ba92d043065aaf43e2a4d74aa00d331e32136a195558dc780ae124c8b1e7b2b2969f3cff8169635df7470f876 SHA512 cab11292ab1957b1682e0d3ee1ddd8e794c52bba7cf08ed39e8f598d3549247c20214d907d1e3792c06a36d8bea55c0bd25ce9aa3a802c262f9af785da7cb340
 DIST seafile-client-7.0.9.tar.gz 2367747 BLAKE2B 60a488a22e094d4783c7da50e8852d7204e16791613d113f729bacc21a40883edc25469f8ce1a8dab52b4ed0ea73b691a831fb0c01a885aeeae7bd276e2781ff SHA512 665a252a321744cd84b96230f5318528aad23a0a089e5049913c34a6c254356c7cc996e45e2a50d432d114f13b089b8d12062b3a1a4d1a0b6f1cbf9688ad32b6
+DIST seafile-client-8.0.1.tar.gz 2388266 BLAKE2B 758862bbf95eb7fe65d0a99b1e8cc790002ad5f4dc47b80c2b612a21ab5439e6ae75337602345a199f3bb6dd81f3d84ac48f7172eac035fca1bd5749a6adb7e2 SHA512 d4300b58253a0f05993f5062080aab622320d685e49eef873961b46171d5b230ff7e5c6a83384489f0c8de115b6eddd92b0d25ae1de6cdd74e2446195ea138c1
```

Comment:
I got this, 
```
>>> Emerging (1 of 1) net-misc/seafile-client-8.0.1::testworld
>>> Downloading 'https://github.com/haiwen/seafile-client/archive/v8.0.1.tar.gz'
--2021-01-15 07:35:57--  https://github.com/haiwen/seafile-client/archive/v8.0.1.tar.gz
Resolving github.com... 140.82.121.4
Connecting to github.com|140.82.121.4|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/haiwen/seafile-client/tar.gz/v8.0.1 [following]
--2021-01-15 07:35:58--  https://codeload.github.com/haiwen/seafile-client/tar.gz/v8.0.1
Resolving codeload.github.com... 140.82.121.9
Connecting to codeload.github.com|140.82.121.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: '/var/cache/distfiles/seafile-client-8.0.1.tar.gz.__download__'

/var/cache/distfile     [          <=>       ]  11.14M  2.56MB/s    in 4.7s    

2021-01-15 07:36:03 (2.35 MB/s) - '/var/cache/distfiles/seafile-client-8.0.1.tar.gz.__download__' saved [11682028]

!!! Fetched file: seafile-client-8.0.1.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      11682028
!!! Expected: 2388266
Refetching... File renamed to '/var/cache/distfiles/seafile-client-8.0.1.tar.gz._checksum_failure_.h1kkg08h'

!!! Couldn't download 'seafile-client-8.0.1.tar.gz'. Aborting.
 * Fetch failed for 'net-misc/seafile-client-8.0.1', Log file:
 *  '/var/tmp/portage/portage/net-misc/seafile-client-8.0.1/temp/build.log'

>>> Failed to emerge net-misc/seafile-client-8.0.1, Log file:
```

I tried regenerating the manifest, but that lead to:
```
>>> Compiling source in /var/tmp/portage/portage/net-misc/seafile-client-8.0.1/work/seafile-client-8.0.1 ...
 * Working in BUILD_DIR: "/var/tmp/portage/portage/net-misc/seafile-client-8.0.1/work/seafile-client-8.0.1_build"
ninja -v -j16 -l16
ninja: error: '/var/tmp/portage/portage/net-misc/seafile-client-8.0.1/work/seafile-client-8.0.1/images/mac/svg/seafile_auto_sync_disabled_white.svg', needed by 'qrc_seafile-client.cpp', missing and no known rule to make it
 * ERROR: net-misc/seafile-client-8.0.1::testworld failed (compile phase):
 *   ninja -v -j16 -l16 failed
 * 
 * Call stack:
 *     ebuild.sh, line  125:  Called src_compile
 *   environment, line 2000:  Called cmake_src_compile
 *   environment, line  686:  Called cmake_build
 *   environment, line  655:  Called eninja
 *   environment, line 1107:  Called die
 * The specific snippet of code:
 *       "$@" || die "${nonfatal_args[@]}" "${*} failed"
```

I guess upstream changed something in their release tarball.

Also looking at issues with 8.0.1, is it safe to ship to users?
https://github.com/haiwen/seafile/issues/2415
Or did they actually fix this with re-shipping seafile-client?

Reply:
About the upstream open issue, it does only concern server packages (seafile-server/seahub).

Reply:
I guess you are waiting that I generate a new Manifest file